extensions/functions_aggregate_generic.yaml

```diff
@@ -0,0 +1,14 @@
+%YAML 1.2
+---
+aggregate_functions:
+  - name: "count"
+    description: Count a set of values
+    impls:
+      - args:
+          - options: [ SILENT, SATURATE, ERROR ]
+            required: false
+          - value: any
+        nullability: DECLARED_OUTPUT
+        decomposable: MANY
```

Comment:
Is the meaning of `decomposable` written down anywhere?

Reply:
Yes: https://github.com/substrait-io/substrait/blob/main/site/docs/expressions/aggregate_functions.md?plain=1#L12